src/esp/gfx/MeshVisualizerDrawable.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+#include <Magnum/Shaders/MeshVisualizer.h>
+#include "Drawable.h"
+#include "esp/gfx/Drawable.h"
+#include "esp/gfx/ShaderManager.h"
+
+namespace esp {
+namespace gfx {
+
+class MeshVisualizerDrawable : public Drawable {
+ public:
+  /**
+   * @brief Constructor
+   *
+   * @param node   Node, to which the drawable is attached
+   * @param shader Shader for the mesh visualizer
+   * @param mesh   Mesh to draw when on render.
+   * @param group  Drawable group this drawable will be added to.
+   */
+  explicit MeshVisualizerDrawable(scene::SceneNode& node,
+                                  Magnum::Shaders::MeshVisualizer3D* shader,
```

Comment:
```suggestion
                                  Magnum::Shaders::MeshVisualizer3D& shader,
```

Why not a reference? The pointer isn't tested for null anywhere inside so I don't think it can be null.